src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -778,7 +778,9 @@ private function preBoot(): ContainerInterface
             $this->startTime = microtime(true);
         }
         if ($this->debug && !isset($_ENV['SHELL_VERBOSITY']) && !isset($_SERVER['SHELL_VERBOSITY'])) {
-            putenv('SHELL_VERBOSITY=3');
+            if (function_exists('putenv')) {
```

Comment:
For the optimization to apply, this must be a qualified call, not a call going through the global fallback. So you need the `\` that was in my previous review (btw, fabbot also asks to add it)